src/cephadm/cephadm

```diff
@@ -985,6 +985,34 @@ def check_ip_port(ctx, ip, port):
         except OSError as e:
             raise Error(e)
 
+def _parse_ss_lpnt(data:str):
+    # ss doen't support json output. Thus we have to parse it.
+    # https://unix.stackexchange.com/questions/157823/list-ports-a-process-pid-is-listening-on-preferably-using-iproute2-tools
+    out = []
+    for line in data.splitlines():
+        if line.startswith('STATE'):
+            if ' '.join(line.split()) != 'State Recv-Q Send-Q Local Address:Port Peer Address:Port':
+                logger.warning('ss -lpnt: unable to parse header "{line}"')
+                return {}
```

Comment:
use the `-H, -no-header` option?

Reply:
how do I then verify that the 4th column is the local address port?